### ISSUE TITLE
Lower r1, r5, r10 and MAP results 

### ISSUE THREAD

**Issue by heerduo** [KovenYu/MAR#33]
Thank for your work. I set batchz-size is 100 or 184, and lr is 2e-5 or 1.414e-4, I got r1 0.000, r5 0.148 r10 0.208, map 5.395. And I have load the pre-training model.

**Comment by mokizzz** [comment]
Mine are better than you. I got r1 30, map 15, and train from scratch. (batch_size=96, lr=2e-4, Duke->Market)

**Comment by KovenYu** [comment]
Sorry for the late reply. What if using the default setting? If that's okay, could you please modify one parameter at a time, so that we can locate where the problem is? Also, did you observe NAN loss like [this issue](https://pan.baidu.com/s/1O0s_dJcbkku6T0MwlLQecw#list/path=%2F)?

**Comment by DeepLearner415** [comment]
Hey I tried to run the demo code, it kept saying that 'cant find data/ml_Market.dat',  I made sure I downloaded all the datasets and other files and saved in the correct directory. any advice ? Thanks
